main/ControllersPresenters/InterviewLevel.java

```diff
@@ -34,22 +33,63 @@ public String getInterviewOutput(String input) throws Exception{
             return getEndOfInterviewPrompt();
         }
         if (currentInterviewMaker.getChoiceOptions().contains(input)) {
-            //check if this is the last thing the intern says
-            //if it's not:
-                // return the corresponding interviewee's next response and next choice options to the player
-            //if it is:
-                //return the interviewee's final response, and prompt player to make hiring decision.
-                //this.getHiringResponse(input);
+            //if the current InterviewIntern has not yet said their last response:
+            if (! this.currentInterviewMaker.checkInternsLastResponse(input)){
+                return GamePrompts.INTERN_RESPONSE_PROMPT +
+                        currentInterviewMaker.displayInternChoiceResponse(input) +
+                        GamePrompts.NEXT_CHOICE_PROMPT +
+                        currentInterviewMaker.getChoiceOptions();
+            }
+
+            //if the current InterviewIntern has said their last response (i.e. at the end of the interview):
+            if (this.currentInterviewMaker.checkInternsLastResponse(input)){
+                return GamePrompts.HIRE_INTERN + this.currentInterviewMaker.internToHire(input);
+            }
+
         }
         if (Objects.equals(input, "yes") || (Objects.equals(input, "no"))){
             //hired the intern/ or not
             //return "successfully/ don't hired intern"
-            //check if there are any interns left to interview haveInterviewsLeft()
-            // if yes: update the current interviewing intern, then also add the next interviewee's next sentenct to output
-            // if no: endLevel() and return hired successful or not
-            currentInterviewMaker.hireIntern();//the actual hiring
+            if (Objects.equals(input, "yes")){
+                this.currentInterviewMaker.hireIntern();
+                //check if there are any interns left to interview haveInterviewsLeft()
+                //if there are more interns to interview, update the current interviewing intern, prompt the player from the
+                // next interviewee's output
+                if (this.currentInterviewMaker.haveInterviewsLeft()){
+                    this.currentInterviewMaker.updateInterviewIntern();
+                    return GamePrompts.HIRED_INTERN + GamePrompts.NEXT_INTERVIEW_INTERN_PROMPT +
+                            this.currentInterviewMaker.getInterviewInternInfo();
+                }
+                // if there are no more interns to interview, endLevel() and return hired successful or not and the
+                // names of all interns they have hired in this interview period.
+                if(! this.currentInterviewMaker.haveInterviewsLeft()){
+                    updateLevelStatus();
```

Comment:
Where is this method updateLevelStatus? I can't seem to find what it does.

Reply:
its at the very bottom of interviewLevel, it updates the level status to end

Reply:
sorry still don't see it. can you tell me the line it starts in?

Reply:
no worries! on line 127

Reply:
thank you! gotcha, this part looks good then